with-cockroachdb/README.md

```diff
@@ -0,0 +1,181 @@
+
+6. The CockroachDB CLI will respond with the following message. In the next section, you can use this console to populate the CockroachDB database.
+
+```bash
+#
+# Welcome to the CockroachDB SQL shell.
+# All statements must be terminated by a semicolon.
+# To exit, type: \q.
+#
+```
+
+Your CockroachDB cluster is now available on the cloud provider you previously selected.
+
+### Populate with data
+
+1. Open the console of the CockroachDB CLI, as shown in the last step of the previous section. In this console, you need to copy-paste the contents on the file `init.sql` from this repository.
+
+```sql
+CREATE TABLE "customer" (
+    email character(50) NOT NULL,
+    id integer NOT NULL,
+    name character(50) NOT NULL
+);
+
+CREATE TABLE "order" (
+    id integer NOT NULL,
+    createdat date NOT NULL,
+    shippingcost double precision,
+    customerid integer NOT NULL,
+    carrier character(20),
+    trackingid character(30)
+);
+
+INSERT INTO
+    customer (email, id, name)
+VALUES
+    ('lucas.bill@example.com', 1, 'Lucas Bill'),
+    /* other */
+
+INSERT INTO
+    "order" (
+        id,
+        createdat,
+        shippingcost,
+        customerid,
+        carrier,
+        trackingid
+    )
+VALUES
+    (1, '2020-08-05', 3, 4, NULL, NULL),
+    /* other */
+```
+
+2. You'll get a response from the CockroachDB CLI stating if your commands were successful:
+
+```bash
+CREATE TABLE
+
+Time: 37ms total (execution 10ms / network 27ms)
+
+CREATE TABLE
+
+Time: 37ms total (execution 10ms / network 27ms)
+
+INSERT 10
+
+Time: 48ms total (execution 18ms / network 30ms)
+
+INSERT 20
+
+Time: 42ms total (execution 14ms / network 29ms)
+```
+
+3. To get an overview of all the tables in your database, run the `SHOW TABLES;` command. This will respond with the following:
+
+```
+  schema_name | table_name | type  | owner      | estimated_row_count | locality
+--------------+------------+-------+------------+---------------------+-----------
+  public      | customer   | table | <USERNAME> |                  10 | NULL
+  public      | order      | table | <USERNAME> |                  20 | NULL
+(2 rows)
+
+Time: 86ms total (execution 20ms / network 66ms)
+```
+
+With the data added to the CockroachDB database, you can set up a connection with StepZen to generate the GraphQL API.
+
+### Copy credentials
+
+To connect StepZen and CockroachDB, you need to get your connection string for the cluster. You can find this by opening your cluster in the CockroachDB dashboard and pressing the "Connection" button (as explained in [Create an account and cluster](#create-an-account-and-cluster)).
+
+![Create CockroachDB instance](./assets/setup-cockroachdb-4.png)
+
+You can copy the file `sample.config.yaml` to a new `config.yaml` file:
+
+```bash
+cp stepzen/sample.config.yaml stepzen/config.yaml
+```
+
+Replace the contents of `config.yaml` with your instance configuration:
+
+```yaml
+# stepzen/config.yaml
```

Comment:
This was actually in the `/cockroachdb` folder, so I got a little messed up here.